tests/e2e/test_main.py

```diff
@@ -832,23 +836,35 @@ def test_data_cp_from_cloud_to_local(
     bucket: str,
     archive_extension: str,
     extract: bool,
+    use_tmp_dir: bool,
 ) -> None:
     TEMP_UNPACK_DIR.mkdir(parents=True, exist_ok=True)
```

Comment:
hmm this test became too complicated with `if extract` / `if not extract` logic. We'll refactor it in future